openwisp_users/apps.py

```diff
@@ -130,6 +137,22 @@ def connect_receivers(self):
             dispatch_uid='make_first_org_user_org_owner',
         )
 
+    @classmethod
+    def handle_organization_update(cls , instance , sender,**kwargs)  :
+        Organization = load_model("openwisp_users" , "Organization")
+        OrganizationUsers = load_model("openwisp_users" ,"OrganizationUser")
+        try:
+            old_obj = Organization.objects.get(pk=instance.pk)
+        except ObjectDoesNotExist:
```

Comment:
```suggestion
        except Organization.DoesNotExist:
```

Reply:
Hey sorry for all that inconsistency should have read the guidelines.
I make these changes and make another commit.

Reply:
No worries, ping me in the [dev chat](https://openwisp.org/support.html) when it's ready for review again.